recipes/salt/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "2015.8.7" %}
+
+package:
+  name: salt
+  version: {{ version }}
+
+source:
+  fn: salt-{{ version }}.tar.gz
+  url: https://github.com/saltstack/salt/archive/v{{ version }}.tar.gz
+  sha256: 93670d3a280e4a4d32fa12e2d5aaaa2ac7cb309c8f7fafff4dd8130fba7b40ff
+
+build:
+  number: 0
+  skip: True  # [win32 or py3k]
+
+requirements:
+  build:
```

Comment:
I made a small patch on the `setup.py` to not download those dll files and everything seams to be working. Thanks for the feedback!